stylus/components/button.styl

```diff
@@ -348,3 +350,109 @@ $button--full
 
 $button--narrow
     min-width auto
+
+/*------------------------------------*\
+  Action button
+\*------------------------------------*/
+$actionbtn
+    @extend $button
+    border-color silver
+    text-transform none
```

Comment:
Why is it necessary ? Isn't it the default ?

Reply:
nope, because I `@extend $button` which overwrite the default value :)